src/components/Event/NewEvent.tsx

```diff
@@ -1,11 +1,51 @@
-import React from 'react'
+import React from "react";
+import Button from "../Button";
 
 const NewEvent = () => {
+  const createNewEvent = () => {};
   return (
-    <>
-        
-    </>
-  )
+    <form
+      onSubmit={createNewEvent}
+      className="flex h-full w-full flex-col items-center gap-5 sm:w-2/5"
+    >
+      <h1 className="text-3xl font-bold text-blue-800">New Event</h1>
+      <TextInput inputType="text" id="name" label="Name of the Event" />
+      <TextInput
+        inputType="textArea"
+        id="description"
+        label="Describe your event:"
+      />
+      <Button type="submit" outline="primary">Create</Button>
+    </form>
+  );
+};
+
+interface TextInputProps {
+  inputType: "text" | "textArea";
+  label: String;
+  id: string;
 }
 
-export default NewEvent
\ No newline at end of file
+const TextInput = (props: TextInputProps) => {
```

Comment:
Should we make it a component so we can use it in other forms?

Reply:
Yeah, I'll pull it out.